mw-webapp/src/logic/homePage/HomePage.tsx

```diff
@@ -181,6 +181,7 @@ export const HomePage = observer(() => {
             <Link
               path={LanguageService.home.whatWeAreBlock.manifestLink[language]}
               className={styles.whatWeAreLink}
+              newTab
```

Comment:
we are using is prefix for boolean variables and props

Reply:
fixed